skabase/SKAAlarmHandler/SKAAlarmHandler.py

```diff
@@ -4,31 +4,29 @@
 #
 #
 #
-
 """ SKAAlarmHandler
 
 A generic base device for Alarms for SKA. It exposes SKA alarms and SKA alerts as TANGO attributes.
 SKA Alarms and SKA/Element Alerts are rules-based configurable conditions that can be defined over multiple
 attribute values and quality factors, and are separate from the "built-in" TANGO attribute alarms.
 """
 # PROTECTED REGION ID(SKAAlarmHandler.additionnal_import) ENABLED START #
-#standard import
+# Standard imports
 import os
 import sys
 from future.utils import with_metaclass
 
-# tango imports
+# Tango imports
 from tango import DebugIt
 from tango.server import run, DeviceMeta, attribute, command, device_property
 
-# SKA specific imports\
+# SKA specific imports
 from skabase import release
-
 file_path = os.path.dirname(os.path.abspath(__file__))
 basedevice_path = os.path.abspath(os.path.join(file_path, os.pardir)) + "/SKABaseDevice"
 sys.path.insert(0, basedevice_path)
 from SKABaseDevice import SKABaseDevice
-# PROTECTED REGION END #    //  SKAAlarmHandler.additionnal_import
+# PROTECTED REGION END #    //  SKAAlarmHandler.additionnal_imports
```

Comment:
This should not have changed - see line 13.  
(These lines are used by POGO, but since its usage seems to be abandoned, won't make much difference).

Reply:
Ok. That was a typo. Will revert the change.